package.json

```
@@ -0,0 +1,34 @@
+{
+  "name": "venmic",
+  "license": "MIT",
+  "author": {
+    "name": "Curve",
+    "url": "https://github.com/Curve"
+  },
+  "version": "2.0.0",
```

since the previous version was just a prerelease, 1.0.0 should be fine
```suggestion
  "version": "1.0.0",
```